src/main/java/edu/iis/powp/factory/FactoryRectangle.java

```diff
@@ -1,6 +1,19 @@
 package edu.iis.powp.factory;
 
+import java.util.ArrayList;
+import edu.iis.powp.command.*;
 
 public class FactoryRectangle {
-
-}
+	private static ArrayList<PlotterCommand> plotterCommands = new ArrayList<>();
+	
+	public  static ComplexCommand createRectangle (int startX, int startY, int width, int height) {
+		
+		plotterCommands.add(new CommandSetPosition(startX,startY));
```

Comment:
Second call to createRectangle will also return commands generated during the first call.
Implementation is poorly tested